api/lib/domain/events/handle-complementary-certifications-scoring.js

```diff
@@ -102,6 +106,7 @@ async function _getComplementaryCertificationResultInfo({
       return {
         isAcquired: true,
         computedComplementaryCertificationBadgeId: lowerLevelComplementaryCertificationBadge.id,
+        lowerLevelComplementaryCertificationBadgeAcquired: true,
```

Comment:
```suggestion
        isLowerLevelComplementaryCertificationAcquired: true,
```
Deux re-nommages proposés dans cette suggestion 💡 
Le is pour marquer le booléen 
Le retrait de `Badge`, (je suis moins sûr), il me semble que sémantiquement, en certif, on obtient la certification complémentaire. 
C'est en positionnement qu'on obtient le badge 🤔

Reply:
Bien vu pour le `is` c'est un oubli 😅 Et je suis aligné pour la proposition de wording.